kernel/kernel-impl/src/main/java/org/sakaiproject/user/impl/BaseUserDirectoryService.java

```diff
@@ -1963,6 +1963,42 @@ public UserEdit newUser(String eid)
 		return u;
 	}
 
+	public boolean updateUserId(String id,String newEmail)
+	{
+		try {
```

Comment:
We should check that the newEmail isn't ever "postmaster" as that bypasses security checks.